Sources/ApodiniDatabase/Context/TypeContainer.swift

```diff
@@ -0,0 +1,186 @@
+            self = .float(value)
+        } else if let value = wrappedValue as? UUID {
+            self = .uuid(value)
+        } else if let value = wrappedValue as? Bool {
+            self = .bool(value)
+        } else if let value = wrappedValue as? String {
+            self = .string(value)
+        } else {
+            self = .noValue
+        }
+    }
+    
+    // swiftlint:disable cyclomatic_complexity
+    func typed() -> Codable? {
+        switch self {
+        case .bool(let value):
+            return value
+        case .string(let value):
+            return value
+        case .int(let value):
+            return value
+        case .int8(let value):
+            return value
+        case .int16(let value):
+            return value
+        case .int32(let value):
+            return value
+        case .int64(let value):
+            return value
+        case .uint(let value):
+            return value
+        case .uint8(let value):
+            return value
+        case .uint16(let value):
+            return value
+        case .uint32(let value):
+            return value
+        case .uint64(let value):
+            return value
+        case .uuid(let value):
+            return value
+        case .float(let value):
+            return value
+        case .double(let value):
+            return value
+        case .noValue:
+            return nil
+        }
+    }
+    
+    // swiftlint:disable cyclomatic_complexity
+    func encode(to encoder: Encoder) throws {
+        var container = encoder.singleValueContainer()
+        switch self {
+        case .bool(let value):
+            try container.encode(value)
+        case .string(let value):
+            try container.encode(value)
+        case .int(let value):
+            try container.encode(value)
+        case .int8(let value):
+            try container.encode(value)
+        case .int16(let value):
+            try container.encode(value)
+        case .int32(let value):
+            try container.encode(value)
+        case .int64(let value):
+            try container.encode(value)
+        case .uint(let value):
+            try container.encode(value)
+        case .uint8(let value):
+            try container.encode(value)
+        case .uint16(let value):
+            try container.encode(value)
+        case .uint32(let value):
+            try container.encode(value)
+        case .uint64(let value):
+            try container.encode(value)
+        case .uuid(let value):
+            try container.encode(value)
+        case .float(let value):
+            try container.encode(value)
+        case .double(let value):
+            try container.encode(value)
+        case .noValue:
+            break
+        }
+    }
+
+    var description: String {
+        self.typed().debugDescription
+    }
+}
+
+extension TypeContainer: LosslessStringConvertible {
+    public init?(_ description: String) {
+        self = .noValue
```

Comment:
What is this used for? What implications does it have that you always return `.noValue`?

Reply:
Since all query parameters need to conform to `LosslessStringConvertible`, but are then internally rated as `.lightweight` and there for not initialised using `init(_ description: String)`, there is currently no need to implement some decision pattern here to init the right type.
But I see your point, this should be handled eventually.

Reply:
Ok, just add a comment with this rationale to the code please.